crates/eips/src/eip4844/builder.rs

```diff
@@ -434,6 +436,17 @@ mod tests {
         assert_eq!(decoded, data);
     }
 
+    #[test]
+    fn big_ingestion_strategy() {
+        let data = vec![1u8; 126_945];
```

Comment:
Blob has size of `128 * 1024` bytes. First 32 bytes are used as length, thus leaving 4095 FEs / 4096 FEs. This constant was calculated as `128 * 1024 - 32 - 4095` (`4095` zeros in each FE).